src/core/pusher.ts

```diff
@@ -180,18 +180,28 @@ export default class Pusher {
     }
   }
 
-  bind(event_name : string, callback : Function) : Pusher {
-    this.global_emitter.bind(event_name, callback);
+  bind(event_name : string, callback : Function, context? : any) : Pusher {
+    this.global_emitter.bind(event_name, callback, context);
     return this;
   }
 
-  unbind(event_name: string, callback: Function) : Pusher {
+  unbind(event_name? : string, callback? : Function, context? : any) : Pusher {
```

Comment:
`context` should probably not be passed here, unless you were going to do something with it. The two `unbind` functions appear to be inconsistent since one handles context and the other does not (before and in this PR).